domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -1857,13 +2240,12 @@ public void executeTrans(TestContext context) {
 
   @Test
   public void executeTransSyntaxError(TestContext context) {
-    Async async = context.async();
     postgresClient = postgresClient();
     postgresClient.startTx(trans -> {
```

Comment:
Only if ALL async callbacks have a context.asyncAssertXXX we don't need a context.async().

Reply:
? don't see it in that function, but a lot of functions could be change to use Async less often.

Reply:
This unit test will call postgresClient.startTx and then successfully complete. It does not wait until startTx has returned some result by calling the callback.
If you want to remove Async you need to add context.asyncAssertXXX to _every_ callback handler:
* postgresClient.startTx
* postgresClient.execute